src/doc/contrib/src/process/release.md

```diff
@@ -127,11 +127,35 @@ module].
 
 ## crates.io publishing
 
-Cargo's library is published to [crates.io] as part of the stable release
-process. This is handled by the [Release team] as part of their process. There
-is a [`publish.py` script] that in theory should help with this process. The
-test and build tool crates aren't published.
-
+Cargo's library and its related dependencies (like `cargo-util`) are published
+to [crates.io] as part of the 6-week stable release process by the [Release
+team]. There is a [`publish.py` script] that is used by the Release team's
```

Comment:
from the release branch, correct?

Reply:
I think yes. This is the same release process as `cargo`'s.

Reply:
To be more specific, it is the `stable` branch in `rust-lang/rust`. I have added some text to try to clarify this.